File: JS/Num_10.js
function solution(arr1, arr2) {
    var answer = [];
    for(i = 0; i < arr1.length; i++)//arr1,arr2 행과 열의 값에 접근하기위해 이중 for문을 씀
        {
            let sum = []; //행렬의 값을 더 해서 저장하기 위해 배열 선언
            for(j = 0; j < arr1[i].length; j++)//각 행열의 0번째와 1번째의 값을 각각 더하기 위해서
                {
                    sum.push(arr1[i][j] + arr2[i][j])
                   // console.log(sum)
                }
            answer.push(sum);
        }
    return answer;
}

File: C/Num_10.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

// sides_len은 배열 sides의 길이입니다.
int solution(int sides[], size_t sides_len) {
    int answer = 0;
    int max = 0;
    int sum = 0;
    int temp = 0;
    for(int i = 0; i<sides_len; i++)
    {
        for(int j = 0; j<sides_len - 1; j++)
            if(sides[i] < sides[j])
            {
                temp = sides[i];
                sides[i] = sides[j];
                sides[j] = temp;
            }
    }
    for(int i = 0; i<sides_len; i++)
    {
        if(i < sides_len - 1)
            sum += sides[i];
        if(i == sides_len - 1)
            max = sides[i];
    }
    printf("max : %d sum : %d",max,sum);
    if(max > sum)
        answer = 2;
    else if(max == sum)
        answer = 2;
    else
        answer = 1;
    return answer;
}